src/qml/main.qml

```diff
@@ -42,12 +60,14 @@ Kirigami.ApplicationWindow {
 			kaidan.connectionStateDisconnected.disconnect(openLoginPage);
 			// open login page
 			pageStack.pop(); // pop all pages
+			applicationWindow().controlsVisible = false; // disable kirigami controls, for disabling the drawer on Login
 			pageStack.push(loginPage, {"isRetry": true});
 		}
 
 
 		if (kaidan.newLoginNeeded()) {
 			// open login page and get new data from user
+			applicationWindow().controlsVisible = false; // disable kirigami controls, for disabling the drawer on Login
```

Comment:
Let's keep this uniformly; only use:
`controlsVisible = false;`

... or add the `applicationWindow()....` everywhere.